apps/web/src/services/asset-inventory/collector/shared/collector-forms/CollectorVersionForm.vue

```diff
@@ -88,8 +88,12 @@ const getVersions = async (pluginId: string) => {
 
 const initSelectedVersion = () => {
     if (!collectorFormState.version.length) {
-        setForm('version', state.versions[0]?.name);
-        collectorFormStore.setVersion(state.versions[0]?.name ?? '');
+        const originAutoUpgrade = collectorFormState.originCollector?.plugin_info?.upgrade_mode === 'AUTO';
```

Comment:
suggestion: How about `isOriginAutoUpgrade` or `isAutoUpgrade`? or else...

Reply:
Hm, I think if it must be changed, every state must be changed.
Otherwise, the same naming will not not confusing. 🤔